gobblin-compaction/src/main/java/gobblin/compaction/CompactionRunner.java

```diff
@@ -75,7 +83,9 @@ public static void main(String[] args) throws ConfigurationException, IOExceptio
       }
     }
     LOG.info("Found " + numOfJobs + " compaction tasks.");
-    PrintWriter pw = new PrintWriter(properties.getProperty(TIMING_FILE, TIMING_FILE_DEFAULT));
+    PrintWriter pw = new PrintWriter(
+        new OutputStreamWriter(new FileOutputStream(properties.getProperty(TIMING_FILE, TIMING_FILE_DEFAULT)),
+            Charset.defaultCharset()));
```

Comment:
OK, I thought we were using the default encoding anyway. I will fix it in another pull request.